src/pages/FocusPage.tsx

```diff
@@ -238,12 +229,34 @@ export const FocusPage = ({
         {variant.name && variant.mutations.length === 0 && (
           <GridCell minWidth={800}>
             <ArticleListWidget.ShareableComponent
-              title='Publications and Pre-Prints'
+              title='Publications and pre-Prints'
               pangolinLineage={variant.name}
             />
           </GridCell>
         )}
       </PackedGrid>
-    </>
+      {variant.mutations.length > 0 && (
+        <p style={{ marginBottom: '30px' }}>
+          The following plots show sequences matching <b>{Math.round(matchPercentage * 100)}%</b> of the
```

Comment:
It doesn't make sense to have this sentence as-is at the bottom of the page. Either move it to the top or change the text "following plots".

Reply:
Ah, good catch. This sentence should be on top as it is essential to the definition of the filter.